src/main/java/no/nav/fo/veilarbdialog/domain/KontorsperreEnhetData.java

```diff
@@ -0,0 +1,10 @@
+package no.nav.fo.veilarbdialog.domain;
+
+/**
+ * Simplifies {@code AuthService}.
+ *
+ * @see no.nav.fo.veilarbdialog.auth.AuthService#filterKontorsperre(KontorsperreEnhetData)
+ */
+public interface KontorsperreEnhetData {
```

Comment:
Fikk lyst til å si at den burde hete Kontorsperrable, men det er kanskje for drøyt